services-camel/src/main/java/edu/unc/lib/cdr/ReplicationDestinationUnavailableException.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.cdr;
+
+public class ReplicationDestinationUnavailableException extends ReplicationException {
+	private static final long serialVersionUID = 8L;
+	
```

Comment:
Does anyone think we may want to overload the constructor to allow this exception to be thrown without a message? (Genuine question, I don't have an opinion on this.)